README-zh-Hans.md

```diff
@@ -1,1792 +1,353 @@
-> * 原文地址：[github.com/donnemartin/system-design-primer](https://github.com/donnemartin/system-design-primer)
-> * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
-> * 译者：[XatMassacrE](https://github.com/XatMassacrE)、[L9m](https://github.com/L9m)、[Airmacho](https://github.com/Airmacho)、[xiaoyusilen](https://github.com/xiaoyusilen)、[jifaxu](https://github.com/jifaxu)、[根号三](https://github.com/sqrthree)
-> * 这个 [链接](https://github.com/xitu/system-design-primer/compare/master...donnemartin:master) 用来查看本翻译与英文版是否有差别（如果你没有看到 README.md 发生变化，那就意味着这份翻译文档是最新的）。
-
-*[English](README.md) ∙ [日本語](README-ja.md) ∙ [简体中文](README-zh-Hans.md) ∙ [繁體中文](README-zh-TW.md) | [العَرَبِيَّة‎](https://github.com/donnemartin/system-design-primer/issues/170) ∙ [বাংলা](https://github.com/donnemartin/system-design-primer/issues/220) ∙ [Português do Brasil](https://github.com/donnemartin/system-design-primer/issues/40) ∙ [Deutsch](https://github.com/donnemartin/system-design-primer/issues/186) ∙ [ελληνικά](https://github.com/donnemartin/system-design-primer/issues/130) ∙ [עברית](https://github.com/donnemartin/system-design-primer/issues/272) ∙ [Italiano](https://github.com/donnemartin/system-design-primer/issues/104) ∙ [한국어](https://github.com/donnemartin/system-design-primer/issues/102) ∙ [فارسی](https://github.com/donnemartin/system-design-primer/issues/110) ∙ [Polski](https://github.com/donnemartin/system-design-primer/issues/68) ∙ [русский язык](https://github.com/donnemartin/system-design-primer/issues/87) ∙ [Español](https://github.com/donnemartin/system-design-primer/issues/136) ∙ [ภาษาไทย](https://github.com/donnemartin/system-design-primer/issues/187) ∙ [Türkçe](https://github.com/donnemartin/system-design-primer/issues/39) ∙ [tiếng Việt](https://github.com/donnemartin/system-design-primer/issues/127) ∙ [Français](https://github.com/donnemartin/system-design-primer/issues/250) | [Add Translation](https://github.com/donnemartin/system-design-primer/issues/28)*
-
-# 系统设计入门
-
-<p align="center">
-  <img src="http://i.imgur.com/jj3A5N8.png"/>
-  <br/>
-</p>
-
-## 目的
-
-> 学习如何设计大型系统。
->
-> 为系统设计的面试做准备。
-
-### 学习如何设计大型系统
-
-学习如何设计可扩展的系统将会有助于你成为一个更好的工程师。
-
-系统设计是一个很宽泛的话题。在互联网上，**关于系统设计原则的资源也是多如牛毛。**
-
-这个仓库就是这些资源的**组织收集**，它可以帮助你学习如何构建可扩展的系统。
-
-### 从开源社区学习
-
-这是一个不断更新的开源项目的初期的版本。
-
-欢迎[贡献](#贡献)！
-
-### 为系统设计的面试做准备
-
-在很多科技公司中，除了代码面试，系统设计也是**技术面试过程**中的一个**必要环节**。
-
-**实践常见的系统设计面试题**并且把你的答案和**例子的解答**进行**对照**：讨论，代码和图表。
-
-面试准备的其他主题：
-
-* [学习指引](#学习指引)
-* [如何处理一个系统设计的面试题](#如何处理一个系统设计的面试题)
-* [系统设计的面试题，**含解答**](#系统设计的面试题和解答)
-* [面向对象设计的面试题，**含解答**](#面向对象设计的面试问题及解答)
-* [其它的系统设计面试题](#其它的系统设计面试题)
-
-## 抽认卡
-
-<p align="center">
-  <img src="http://i.imgur.com/zdCAkB3.png"/>
-  <br/>
```

Comment:
Hey, is there any actual reason you're using `<br/>` instead of `<br>`?

Reply:
This can help explaining the reason https://stackoverflow.com/a/1946446

>Simply `<br>` is sufficient.
> The other forms are there for compatibility with XHTML; to make it possible to write the same code as XHTML, and have it also work as HTML.

Reply:
Oh, I see, so it is XHTML. I've never gone further about it... Thanks for explaining!